Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -368,7 +368,7 @@ public void TestRunsetAutoRunConfigCreationContent()
             executer.RunSetConfig = runsetConfig;
             CLIHelper cliHelp = new CLIHelper();
             RunSetAutoRunConfiguration autoRunConfig = new RunSetAutoRunConfiguration(sol, executer, cliHelp);
-
+            WorkSpace.Instance.Solution = sol;
```

Comment:
need instead to pass the Solution as param to RunSetAutoRunConfiguration constructor